src/util.rs

```diff
@@ -134,16 +134,27 @@ pub fn ask(config: &Config, question: &str, default: bool) -> bool {
     }
     let stdin = stdin();
     let mut input = String::new();
-    let _ = stdin.read_line(&mut input);
-    let input = input.to_lowercase();
-    let input = input.trim();
-
-    if input == tr!("y") || input == tr!("yes") {
-        true
-    } else if input.trim().is_empty() {
-        default
-    } else {
-        false
+    match stdin.read_line(&mut input) {
+        Ok(0) => {
+            println!(" -> EOF");
```

Comment:
Pacman doesn't print anything for this. In fact it doesn't even print a new line.

Paru tries to match pacman when possible so negative on EOF is good but I don't think we should print anything but a new line. Otherwise it kind of messes up the input. I'll see about changing pacman to be like that too.

Reply:
I agree that this should be handled in Pacman. 
Thanks for your time reviewing my stuff and giving feedback. 
I will try to get all PRs fixed as soon as possible.